components/txt_werk/txt_werk.app.mjs

```diff
@@ -1,11 +1,53 @@
+import { axios } from "@pipedream/platform";
+import constants from "./common/constants.mjs";
+
 export default {
   type: "app",
   app: "txt_werk",
-  propDefinitions: {},
+  propDefinitions: {
+    text: {
+      type: "string",
+      label: "Text",
+      description: "Text to be analyzed",
+    },
+    title: {
+      type: "string",
+      label: "Title",
+      description: "Title of the document to be analyzed",
+    },
+    services: {
+      type: "string[]",
+      label: "Services",
+      description: "List of services to request",
+      options: constants.SERVICES,
+    },
+  },
   methods: {
-    // this.$auth contains connected account data
-    authKeys() {
-      console.log(Object.keys(this.$auth));
+    _baseUrl() {
+      return "https://api.txtwerk.de/rest/txt";
+    },
+    async _makeRequest(opts = {}) {
+      const {
+        $ = this,
+        path,
+        headers,
+        ...otherOpts
+      } = opts;
+      return axios($, {
+        ...otherOpts,
+        url: this._baseUrl() + path,
+        headers: {
+          ...headers,
+          "X-Api-Key": `${this.$auth.api_key}`,
+        },
+      });
+    },
+    async analyzeText(args = {}) {
+      return this._makeRequest({
+        method: "post",
```

Comment:
This is a POST with query parameters?

Reply:
Yes. ["A solicitação pode ser feita como uma solicitação GET ou POST para a URL"](https://services.txtwerk.de/ws/documentation/showRequest)